test/end-to-end/processor/coverage/_normalizer.xsl

```diff
@@ -47,17 +47,18 @@
 	</xsl:template>
 
 	<!--
-		Normalizes the links to the tested module
+		Normalizes the links to the test, the tested module, and sections of this report
```

Comment:
FYI in case you want to know what this file is for: the test cases under `test/end-to-end` compare expected reports with newly generated reports, and part of the process is to normalize both reports in the pair prior to comparison. This stylesheet's job is to identify what fragments need normalization and what to replace them with. For instance, this stylesheet is why the reports under `test/end-to-end/cases-coverage/expected/stylesheet` have headers like `module: xsl-analyze-string-01.xsl` with only a filename instead of a full file path.